src/main/scala/scalismo/faces/parameters/RenderObject.scala

```diff
@@ -141,6 +143,20 @@ object MoMoInstance {
            modelURI: URI): MoMoInstance = {
     fromCoefficients(MoMoCoefficients.zeros(shapeComponents, colorComponents, expressionComponents), modelURI)
   }
+
+  /** create a random coefficient vector with its elements N(0,1) distributed. */
+  def sample(model: MoMo, modelURI: URI)(implicit rnd: Random): MoMoInstance = {
```

Comment:
The first part of this function should go to MoMoCoefficients.sample() and then call this function here.